dist/preview release/what's new.md

```diff
@@ -337,6 +337,7 @@
 - Fix issue where default free camera rotation didn't work during pointer lock ([PolygonalSun](https://github.com/PolygonalSun))
 - Fix issue with wrong definition of a returned BASIS format ([RaananW](https://github.com/RaananW))
 - Fix glTF exporter exports unused materials from excluded meshes ([daoshengmu](https://github.com/daoshengmu))
+- Fix canvasGraphService formatting. ([carolhmj](https://github.com/carolhmj))
```

Comment:
I don't think you should put an entry in the what's new file for changes in file formatting: else the file would be filled with hundreds/thousands of Raanan changes :)

Reply:
agree!